Examples/Tests/MacroExamples/Implementation/Member/CustomCodableTests.swift

```diff
@@ -0,0 +1,72 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import MacroExamplesImplementation
+import SwiftSyntaxMacros
+import SwiftSyntaxMacrosTestSupport
+import XCTest
+
+final class CustomCodableTests: XCTestCase {
+  private let macros = ["CustomCodable": CustomCodable.self]
+
+  func testExpansionAddsDefaultCodingKeys() {
+    assertMacroExpansion(
+      """
+      @CustomCodable
+      struct Person {
+        let name: String
+        let age: Int
+      }
+      """,
+      expandedSource: """
+        struct Person {
+          let name: String
+          let age: Int
+
+            enum CodingKeys: String, CodingKey {
+                    case name
+                case age
+              }
```

Comment:
You should be able to fix this formatting by changing `CustomCodable` to return

```swift
let codingKeys: DeclSyntax = """
  enum CodingKeys: String, CodingKey {
  \(raw: cases.joined(separator: "\n"))
  }
  """
```

Reply:
Done in https://github.com/apple/swift-syntax/pull/2163/commits/39032ab805de40a7a4c7416a71a3f7704662a8ce. Thanks!